nomos-da-core/reed-solomon/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+use reed_solomon_erasure::{galois_8::ReedSolomon, Error};
+
+/// Reed Sololomon encode the elements with a custom parity ratio
+///  # Arguments
+/// * `parity_ratio` - Ratio of parity elements over original elements size
+/// * `elements` - Elements to encode
+pub fn encode_elements(parity_ratio: usize, elements: &[u8]) -> Result<Vec<u8>, Error> {
+    let mut encoded = vec![vec![0]; elements.len() * (parity_ratio + 1)];
+    for (i, &e) in elements.iter().enumerate() {
+        // review bytes encoding
+        encoded[i] = e.to_be_bytes().to_vec();
```

Comment:
I think that `to_be_bytes()` is good enough just in case when the system is little-endian.

Reply:
I'm not sure if use this or just `vec![e]`. Results are the same, and for our case in where we use direct bytes it may be better, because we know our representation will not go outside of that range. I'll probably do some benchmarking at some point, maybe we can squeeze a bit of performance there.